core/templates/domain/admin/admin-backend-api.service.ts

```diff
@@ -140,8 +140,70 @@ export class AdminBackendApiService {
       });
     });
   }
-}
+  flushCache():Promise<void> {
```

Comment:
This is an async function, can you please change it to `flushCacheAsync`? (Ditto for other functions returning promise)

Reply:
done!